specifications/api/swagger.yaml

```diff
@@ -1427,6 +1427,15 @@ components:
         id: *addressId
         state: *addressState
 
+    ApiAddressInfo: &ApiAddressInfo
+      type: object
+      required:
+        - address
+        - derivation_path
+      properties:
+        address: *ApiAddress
+        derivation_path: *derivationPath
```

Comment:
This breaks the current API in an unfortunate and unnecessary way. We can introduce the extra `derivation_path` in the top-level object, without indirection. Is there any reason why this indirection was introduced? 

It seems that it would suffice to extend `ApiAddress` with a new, required, `derivation_path` field.